types/n-readlines/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for socket.io-file 1.0
```

Comment:
This should be n-readlines.

Side note: How did this pass the CI tests?

Reply:
Ah good spot, I copy pasted this from my previous one, thought I'd changed it all but I guess it was just the version number I did